### PR TITLE
Tbolt/3572 Generate Filename for Certifications

### LINKED ISSUE #3572: [Feature] Generate a filename for the certification letters when they are added
**Description and related issues -**
When state certification letters are viewed or downloaded, they have a generic name. Update the code to generate a name using the format (Name)-(State)-(FFY).pdf

### This task is done when…

- [ ] the pdfs have a name matching the format (Lastname)-(Firstname)-(State)-(FFY).pdf when they are viewed or downloaded

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #3572 

**Description-**
Updates files to have a generated name when downloading them

**Steps to manually verify this change...**

1. Login as fed admin
2. Upload a couple files (pdf and doc/docx would be good)
3. Download those files
4. Verify they download and open as expected
5. Verify the names match the proposed scheme in the ticket

### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [x] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: api/routes/auth/certifications/files/get.endpoint.js [hunk 1]
```diff
@@ -29,8 +29,10 @@ describe('auth/certifications/files endpoints', () => {
 
       it('with a valid request', async () => {
         const response = await api.get(
-          url('74aa0d06-ae6f-472f-8999-6ca0487c494f')
+          url('test-123')
         );
+        
+        console.log("response", response);
```
Comment (thetif) [inline api/routes/auth/certifications/files/get.endpoint.js:35]: can you take out this console log before you merge it?

**Review by thetif — APPROVED** [review]
looks good

**Review by Sun-Mountain — APPROVED** [review]
CONFIRMED:

| Given         | When     | Then   | Result     | 
|--------------|-----------|------------|------------|
| the user downloads a letter | user clicks on `pdf view` icon | the letter downloads and opens as expected | ✅  PASS|
| the letter downloads | user clicks on `pdf view` icon | the names match the proposed scheme on ticket | ✅  PASS|

**Review by akuas — APPROVED** [review]
Works as intended.

**Review by NAretakis — APPROVED** [review]
Approved! Works as advertised, thanks Ty.

**Pull request merged**